curriculum/challenges/english/25-front-end-development/review-javascript-events/6723cc0ca05ce9b87a319ceb.md

```diff
@@ -9,7 +9,40 @@ dashedName: review-javascript-events
 
 Review the concepts below to prepare for the upcoming quiz.
 
+## The Change Event
 
+- **Definition**: The change event is a special event which is fired when the user modifies the value of certain input elements. Examples would include when a checkbox or a radio button is ticked. Or when the user makes a selection from something like a date picker or dropdown menu.
+
+```html
+<label>
+  Choose a programming language:
+  <select class="language" name="language">
+    <option value="">---Select One---</option>
+    <option value="JavaScript">JavaScript</option>
+    <option value="Python">Python</option>
+    <option value="C++">C++</option>
+  </select>
+</label>
+
+<p class="result"></p>
+```
+
+```js 
+const selectEl = document.querySelector(".language");
+const result = document.querySelector(".result");
+
+selectEl.addEventListener("change", (e) => {
+  result.textContent = `You enjoy programming in ${e.target.value}.`;
+});
+```
+
+## Event Bubbling
+
+- **Definition**: Event bubbling, or propagation, refers to how an event “bubbles up” to parent objects when triggered.
+
+## Event Delegation
+
+- **Definition**: Event delegation is the process of taking a captured event, and delegating it to another element. 
```

Comment:
```suggestion
- **Definition**: Event delegation is the process of listening to events that have bubbled up to a parent, rather than handling them directly on the element that triggered them. 
```
It's a bit more verbose, but I think this should clarify exactly event delegation is.